### PR TITLE
MPP-3442: Strings for new forwarded email content

### PR DESCRIPTION
**Pull request by rafeerahman**
New strings for email content.

(https://github.com/mozilla/fx-private-relay/pull/3962)

### CODE REVIEW

File: en/misc.ftl [hunk 1]
```diff
@@ -267,6 +267,20 @@ first-time-user-email-hero-primary-text = As a { -brand-name-firefox } user, you
 first-time-user-email-hero-secondary-text = Manage all your masks from your { -brand-name-relay } dashboard.
 first-time-user-email-hero-cta = View your dashboard
 
+## Email sent to people onboarding that receive their first forwarded email.
+
+our-promise-header = Our promise to you
+our-promise-content = { -brand-name-relay } email masks forward to your email inbox any messages sent to that email mask—without revealing your real email address to the sender.
```
Comment (peiying2) [inline en/misc.ftl:273]: need spaces before and after the M dash.

File: en/misc.ftl [hunk 2]
```diff
@@ -267,6 +267,20 @@ first-time-user-email-hero-primary-text = As a { -brand-name-firefox } user, you
...
+what-can-you-do-with-relay-title = What can you do with { -brand-name-firefox-relay }?
+what-can-you-do-with-relay-point-one = Get emails without revealing your real email address to senders
+what-can-you-do-with-relay-point-one-subpoint-html = Just share an email mask instead. Email masks will forward emails to your inbox. You can create masks directly on { -brand-name-firefox }, with the <a href="{ $url }" { $attrs }>{ -brand-name-relay } extension</a>, or on your { -brand-name-relay } dashboard.
+what-can-you-do-with-relay-point-two = Keep websites and spammers from easily tracking who you are
+what-can-you-do-with-relay-point-two-subpoint = Email trackers are everywhere these days. { -brand-name-relay } email masks help you to keep them guessing.
+what-can-you-do-with-relay-point-three = Block spam from reaching your inbox, and even turn masks off completely
```
Comment (peiying2) [inline en/misc.ftl:279]: I don't understand the second part of the sentence.  Who turns the masks off? I thought you want the masks on?

**Review by peiying2 — CHANGES_REQUESTED** [review]
Looks good overall, only some minor comments.

File: en/misc.ftl [hunk 3]
```diff
@@ -267,6 +267,20 @@ first-time-user-email-hero-primary-text = As a { -brand-name-firefox } user, you
 first-time-user-email-hero-secondary-text = Manage all your masks from your { -brand-name-relay } dashboard.
 first-time-user-email-hero-cta = View your dashboard
 
+## Email sent to people onboarding that receive their first forwarded email.
+
+our-promise-header = Our promise to you
+our-promise-content = { -brand-name-relay } email masks forward to your email inbox any messages sent to that email mask—without revealing your real email address to the sender.
+what-can-you-do-with-relay-title = What can you do with { -brand-name-firefox-relay }?
+what-can-you-do-with-relay-point-one = Get emails without revealing your real email address to senders
+what-can-you-do-with-relay-point-one-subpoint-html = Just share an email mask instead. Email masks will forward emails to your inbox. You can create masks directly on { -brand-name-firefox }, with the <a href="{ $url }" { $attrs }>{ -brand-name-relay } extension</a>, or on your { -brand-name-relay } dashboard.
```
Comment (peiying2) [inline en/misc.ftl:276]: Usually we include a comment where the URL points to, to give localizer some context.

File: en/misc.ftl (hunk 1 above)
Comment (rafeerahman) [reply en/misc.ftl:273]: Do you mean the dash in "mask-without"? The design intends for them to have no space between them

Comment (flodolo) [reply en/misc.ftl:273]: [These are the guidelines](https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o#section-hyphens-and-dashes-c9) from content

Comment (peiying2) [reply en/misc.ftl:273]: If it's connected like that, it's a hyphenation, the word would be a hyphenated word would be `mask-without`. I don't think that's the meaning here.

Comment (rafeerahman) [reply en/misc.ftl:273]: Is using an em-dash here ok? (its already an em dash currently)

Comment (rafeerahman) [reply en/misc.ftl:273]: Nevermind, I understand why said it needs spaces now

**Comment by rafeerahman** [comment]
> Looks good overall, only some minor comments.

Thanks for the comments. Made these changes now.

**Review by peiying2 — APPROVED** [review]

**Pull request merged**